### PR TITLE
remove deprecated Oauth1 tokens

### PR DESCRIPTION
**Pull request by sushithegreat**

### CODE REVIEW

File: contextio/__init__.py [hunk 1]
```diff
@@ -353,12 +353,6 @@ def post_account(self, **params):
...
                 Ignored if any of the provider_* parameters are set below.
-            provider_token: string - An OAuth token obtained from the IMAP 
-                account provider to be used to authentify on this email 
-                account.
-            provider_token_secret: string - An OAuth token secret obtained 
-                from the IMAP account provider to be used to authentify on 
-                this email account.
             provider_refresh_token: An OAuth2 refresh token obtained from the
                 IMAP account provider to be used to authentify on this email
                 account.
```
Comment (bpot) [inline contextio/__init__.py]: authenticate? access? this email account

**Comment by bpot** [comment]
This looks good to me :+1:. Maybe get a second opinion though

**Comment by kerinin** [comment]
:+1:, assuming we're OK with potentially breaking some people's code

**Comment by sushithegreat** [comment]
None of the email providers we work with support Oauth1 anymore, so it shouldn't be a problem.

**Pull request merged**